src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/draftorders/upload/UploadDraftOrdersSubmittedHandler.java

```diff
@@ -56,8 +66,13 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(
             finremCallbackRequest.getEventType(), finremCallbackRequest.getCaseDetails().getId());
 
         FinremCaseDetails caseDetails = finremCallbackRequest.getCaseDetails();
-        String confirmationBody = getConfirmationBody(caseDetails);
+        NotificationRequest judgeNotificationRequest = draftOrdersNotificationRequestMapper.buildJudgeNotificationRequest(caseDetails);
+        notificationService.sendContestedReadyToReviewOrderToJudge(judgeNotificationRequest);
 
+        caseDetails.getData().getDraftOrdersWrapper().setUploadSuggestedDraftOrder(null); // Clear the temporary field
```

Comment:
Does this work? According to the CCD documentation you can't update case data in a submitted handler

Reply:
Working fine locally.

Reply:
yep